src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion.py

```diff
@@ -126,7 +125,7 @@ def __call__(
         if accepts_eta:
             extra_step_kwargs["eta"] = eta
 
-        for i, t in tqdm(enumerate(self.scheduler.timesteps)):
+        for i, t in enumerate(self.progress_bar(self.scheduler.timesteps)):
```

Comment:
As for `enumerate` and `tqdm`, if the order is `enumerate(tqdm(...))`, we don't have to pass `total`.

Reply:
Agree that's what it states in the `tqdm` docs: https://pypi.org/project/tqdm/#faq-and-known-issues